forge-gui/res/cardsfolder/upcoming/threats_undetected.txt

```diff
@@ -0,0 +1,9 @@
+Name:Threats Undetected
+ManaCost:2 G
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Library | Destination$ Library  | ChangeType$ Creature | ChangeNum$ 4 | DifferentCMC$ True | RememberChanged$ True | Reveal$ True | Shuffle$ False | SubAbility$ DBTwoPiles | SpellDescription$ Search your library for up to four creature cards with different powers and reveal them. An opponent chooses two of those cards. Shuffle the chosen cards into your library and put the rest into your hand.
```

Comment:
Different power != different cmc

Reply:
looks like this needs engine support

Reply:
@tool4ever Yes, I'm toying around with changezoneeffect.java trying to create a new param. But this might take some time. Will remove it for now